package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@digitalbazaar/ed25519-verification-key-2020",
-  "version": "2.1.2-0",
+  "version": "3.0.0-0",
```

This should be reverted. The release tools handle it -- if you change this manually the tools may break. Just make sure you update the changelog indicating it will be a major release (and have notes there on what changed).

```suggestion
  "version": "2.1.2-0",
```